test/NuGet.Clients.Tests/NuGet.CommandLine.Test/ConsoleTest.cs

```diff
@@ -11,5 +15,25 @@ public void TestConsoleWindowWidthNotZero()
         {
             Assert.NotEqual(0, new Console().WindowWidth);
         }
+
+        [Theory]
+        [InlineData(2, "abcd\nwxyz", "  abcdXXX  wxyzXXX", 10)]
+        [InlineData(2, "abcd\rwxyz", "  abcdXXX  wxyzXXX", 10)]
+        [InlineData(3, "abcde\n\nwcjb", "   abcdeXXXXXX   wcjbXXX", 10)]
+        [InlineData(2, "abcde\r\nwcjb", "  abcdeXXX  wcjbXXX", 10)]
+        [InlineData(2, "abcde\rwcjb", "  abcdeXXX  wcjbXXX", 10)]
+        [InlineData(2, "abcdefghijk\nAB", "  abcdeXXX  fghijXXX  kXXX  ABXXX", 7)]
+        [InlineData(2, "ab  cd", "  abXXX  cdXXX", 6)]
+        public void TestPrintJustified(int indent, string input, string expected, int width)
```

Comment:
Maybe play around with Cursor position?
Maybe test some spaces at start and/or end? (see my other comment

Reply:
I've added tests for both, and adapted my code to pass them.